python/apps/taiga/src/taiga/base/serializers/fields.py

```diff
@@ -23,5 +23,8 @@ def __get_validators__(cls) -> CallableGenerator:
         yield cls.validate
 
     @classmethod
-    def validate(cls, value: UUID) -> str:
+    def validate(cls, value: UUID | str) -> str:
+        if not isinstance(value, UUID):
+            # this happend when we try to serialize an id twice
+            return value
```

Comment:
The double serialization may be solved (once we get updated) by the change I suggest you by setting the response_model to None. In that case this if would't be required.
May it deserve a quick test?

Reply:
I think this is now required in this PR because we don't have anything about the other PR. Can I change this and response_model to None and  will your user tests work?